platform/android/MapboxGLAndroidSDK/.gitignore

```diff
@@ -0,0 +1,2 @@
+lint-baseline.xml
+lint/lint-baseline-local.xml
```

Comment:
@Guardiola31337 Why do we have different baselines for local dev and for CI? Shouldn't they both be the same?